packages/application/src/shell.ts

```diff
@@ -689,6 +690,27 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     }
   }
 
+  /*
+   * Whether the add buttons for each main area tab bar are enabled.
+   */
+  get addButtonEnabled(): boolean {
+    return this._dockPanel.addButtonEnabled;
+  }
+
+  /*
+   * Set whether the add buttons for each main area tab bar are enabled.
+   */
```

Comment:
The practice is to document only once for getter/setter pair.

```suggestion
```

Reply:
done